docs/DeveloperGuide.md

```diff
@@ -158,103 +165,267 @@ Classes used by multiple components are in the `seedu.address.commons` package.
+
+</div>
+
+---
+
+
+### Delete Meeting with an Udder Feature
+
+#### Feature
+`deletem INDEX` where `INDEX` is the index of the meeting to be deleted.
+
+#### Feature Purpose
+The `deletem` command allows users to delete an existing `Meeting` from the `AddressBook`, impacting the schedule of the `Person` associated with the meeting.
+
+#### Key Components
+- `DeleteMeetingCommand`: Executes the deletion of a `Meeting` from both the `AddressBook` and the `Person`'s `Meetings` based on the user's input.
+- `DeleteMeetingCommandParser`: Parses user input to create a `DeleteMeetingCommand` object.
+- `LogicManager`: Invokes the `DeleteMeetingCommand` to execute the deletion operation.
+- `ModelManager`: Implements the `Model` interface and contains the internal list of persons (Udders) and meetings.
+- `Person`: Represents an Udder in FindingbrUdders, encapsulating their personal information including scheduled meetings.
+- `Index`: Represents the index that refers to the target Udder of the `Meeting` to be created.
+- `Meeting`: Represents a scheduled meeting, containing details such as participants, location, start time, and end time.
+- `Meetings`: Represents an ArrayList that contains `Meeting` objects.
+- `AddressBookParser`: Creates a `DeleteMeetingCommand` object based on the user input.
+
+### Sequence of Action
+To understand how the `deletem` command works, here is a list of steps illustrating what occurs when `LogicManager#execute()` is invoked:
+
+We will be using the user input `deletem 1` as an example.
+
+1. The user inputs the command `deletem 1`, intending to delete the meeting at index `1`.
+2. The `DeleteMeetingCommandParser` interprets the input.
+3. A `DeleteMeetingCommand` object is created.
+4. The `LogicManager` invokes the `execute` method of `DeleteMeetingCommand`.
+5. The `execute` method of `DeleteMeetingCommand` retrieves the existing `Meeting` object to be deleted from the `AddressBook` by `Index`.
+6. The `execute` method of `DeleteMeetingCommand` finds the `Person` object associated with the meeting being deleted.
+7. The `execute` method of `DeleteMeetingCommand` removes the `Meeting` from the `Meetings` in the `AddressBook` and the `Person` object found earlier.
+8. The `execute` method of `DeleteMeetingCommand` returns a `CommandResult` object which stores the data regarding the completion of the `DeleteMeetingCommand`.
+9. The UI updates to reflect this updated list of meetings in the DetailPanel of the Udder if the affected `Person` is being displayed.
+
+<div markdown="span" class="alert alert-info">:information_source: **Note**:
+
+- At step 2, if any arguments are missing, an error will be shown on the screen and the `DeleteMeetingCommand` object will NOT be created!
+
+</div>
+
+---
+
+### Edit Meeting with an Udder Feature
+
+#### Command Feature
+`editm INDEX [n/NAME] [l/LOCATION] [st/START_TIME] [et/END_TIME]` where at least one optional field should be present.
```

Comment:
As per the changes made to `edit` command regarding the optional fields, it can be copy pasted here for consistency